### ISSUE TITLE
Processing of raw datasets

### ISSUE THREAD

**Issue by sainan-zhang** [mks0601/3DMPPE_POSENET_RELEASE#101]
It is a very excellent work! Thanks for your detailed sharing. I have learned your codes of ROOTNET and POSENET and I want to make some improvements based on your networks. But I don't know how you process the original dataset into the form you shared. In your datasets, such as Human36M, the image files are stored in many separate folders. Can I treat the image in each folder as each frame of an action video? Looking forward to your reply! Thanks!

**Comment by mks0601** [comment]
Yes you can treat the image in each folder as each frame of an action video. All information about data are included in the provided json files.

**Comment by sainan-zhang** [comment]
Thanks very much!
